Meta.Tests.ps1

```diff
@@ -12,23 +12,39 @@ $errorActionPreference = 'Stop'
 $testHelperModulePath = Join-Path -Path $PSScriptRoot -ChildPath 'TestHelper.psm1'
 Import-Module -Name $testHelperModulePath
 
-$moduleRootFilePath = Split-Path -Path $PSScriptRoot -Parent
-$moduleName = (Get-Item -Path $moduleRootFilePath).Name
-$dscResourcesFolderFilePath = Join-Path -Path $moduleRootFilePath -ChildPath 'DscResources'
-
 <#
-    This is a workaround to be able to test DscResource.Tests module.
-    Because the name has punctuation in the name, AppVeyor replaces
-    that with a dash when it creates the folder structure, so the folder
-    name becomes 'dscresource-tests'. This sets the module name to the
-    correct name.
-    If the name can be detected in a better way, for DscResource.Tests
-    and all other modules, then this could be removed.
+    This is a workaround to be able to test DscResource.Tests module. When
+    DscResource.Tests is testing itself, we need to determine if we are running
+    the code on repository DscResource.Tests or some other resource module.
+    if the $env:APPVEYOR_BUILD_FOLDER match the regex 'dscresource.tests' the
+    tests is considered that it should run on it self.
+    The regex will match both 'DscResource.Tests', when run locally, and
+    'dscresource-tests' when run in AppVeyor.
+
+    NOTE: To be able to run these common tests locally the environment variable
```

Comment:
Done.

Now it works running the common tests locally for DscResource.Tests module without adding any environment variable.

Tested for "default"-model here; https://ci.appveyor.com/project/johlju/xfailovercluster/build/1.6.243.0#L28
Tested for "Harness"-model here; https://ci.appveyor.com/project/johlju/xstorage/build/2.9.19.5#L1398

And this PR is passing tests like it should.